docs/themes/redwood/assets/sass/uk/_components.scss

```diff
@@ -0,0 +1,99 @@
+/**
+ * Copyright © 2020, Oracle and/or its affiliates. All rights reserved.
+ * The Universal Permissive License (UPL), Version 1.0
+ */
+
+ // Base
+@import "uikit/src/scss/components/variables";
+@import "uikit/src/scss/components/mixin";
+@import "uikit/src/scss/components/base";
+
+// Elements
+@import "uikit/src/scss/components/link";
+@import "uikit/src/scss/components/heading";
+@import "uikit/src/scss/components/divider";
+@import "uikit/src/scss/components/list";
+// @import "uikit/src/scss/components/description-list";
+@import "uikit/src/scss/components/table";
+@import "uikit/src/scss/components/icon";
+// @import "uikit/src/scss/components/form-range";
+@import "uikit/src/scss/components/form"; // After: Icon, Form Range
+@import "uikit/src/scss/components/button";
+
+// Layout
+@import "uikit/src/scss/components/section";
+@import "uikit/src/scss/components/container";
+@import "uikit/src/scss/components/grid";
+@import "uikit/src/scss/components/tile";
+@import "uikit/src/scss/components/card";
+
+// Common
+// @import "uikit/src/scss/components/close"; // After: Icon
+// @import "uikit/src/scss/components/spinner"; // After: Icon
+// @import "uikit/src/scss/components/totop"; // After: Icon
+// @import "uikit/src/scss/components/marker"; // After: Icon
+@import "uikit/src/scss/components/alert"; // After: Close
+// @import "uikit/src/scss/components/badge";
+@import "uikit/src/scss/components/label";
+// @import "uikit/src/scss/components/overlay"; // After: Icon
+// @import "uikit/src/scss/components/article"; // After: Subnav
+// @import "uikit/src/scss/components/comment"; // After: Subnav
+@import "uikit/src/scss/components/search"; // After: Icon
+
+// Navs
+@import "uikit/src/scss/components/nav";
+@import "uikit/src/scss/components/navbar"; // After: Card, Grid, Nav, Icon, Search
+@import "uikit/src/scss/components/subnav";
+@import "uikit/src/scss/components/breadcrumb";
+// @import "uikit/src/scss/components/pagination";
+@import "uikit/src/scss/components/tab";
+// @import "uikit/src/scss/components/slidenav"; // After: Icon
+@import "uikit/src/scss/components/dotnav";
+// @import "uikit/src/scss/components/thumbnav";
+
+// JavaScript
+// @import "uikit/src/scss/components/accordion";
```

Comment:
nit: should these commented out imports be removed?

Reply:
Leaving in place is *much* easier than trying to re-introduce later if we want to include.